src/contracts/Repository/Values/Content/URLWildcard/Query/Criterion/LogicalAnd.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Contracts\Core\Repository\Values\Content\URLWildcard\Query\Criterion;
+
+class LogicalAnd extends LogicalOperator
```

Comment:
```suggestion
final class LogicalAnd extends LogicalOperator
```


Applies to other criterion as well.